src/main/java/uk/gov/ons/fwmt/job_service/data/legacy_ingest/LegacySampleIngest.java

```diff
@@ -167,10 +167,13 @@
 
   private String tmJobId;
   private LocalDate dueDate;
-
+  
+  // // // Added the lat and long values 
+  @CSVColumn("Lat")
   @JobAdditionalProperty("geoX")
```

Comment:
change so we have both lat and long and OSGridref saved as geo x and y from the the file